mkdocs_static_i18n/plugin.py

```diff
@@ -140,6 +132,17 @@ def _get_page_from_paths(self, expected_paths, files):
                 f"mkdocs-static-i18n is expecting one of the following files: {expected_paths}"
             )
 
+    def _fix_dest_paths(self, page):
+        """
+        Corrects paths to work fine on Windows and Unix system
+        """
+        i18n_page = deepcopy(page)
+        i18n_page.dest_path = page.dest_path.replace(str(Path(f"{page.name}/")), "")
```

Comment:
please leave the `replace()` stuff where they are in the code, in this function we only want to Path() them

```suggestion
        i18n_page.dest_path = str(Path(i18n_page.dest_path))
```

Reply:
(same for abs_dest_path)